libraries/Microsoft.Bot.Builder.Dialogs/Constants/DialogPath.cs

```diff
@@ -1,6 +1,37 @@
-﻿namespace Microsoft.Bot.Builder.Dialogs
+﻿using System.Data;
+
+namespace Microsoft.Bot.Builder.Dialogs
 {
     public class DialogPath
     {
+        /// <summary>
+        /// Counter of emitted events.
+        /// </summary>
+        public const string EventCounter = "dialog.eventCounter";
```

Comment:
>dialog.eventCounter [](start = 44, length = 19)

we should consider prefixing these with an _ or something to avoid collisions with user props. Or we may want to define a new internal scope.

Reply:
I left these unprefixed.  I have a discussion thread I started on teams about how to handle this in general.  The little feedback I have is that we should have some reserved words and this is likely one of them.

Reply:
I'm leaving these for now because they are used in trigger conditions.

---
In reply to: [343890945](https://github.com/microsoft/botbuilder-dotnet/pull/2936#discussion_r343890945) [](ancestors = 343890945)